lsm6dsr_reg.c

```diff
@@ -7369,20 +7369,19 @@ int32_t lsm6dsr_batch_counter_threshold_get(const stmdev_ctx_t *ctx,
   */
 int32_t lsm6dsr_fifo_data_level_get(const stmdev_ctx_t *ctx, uint16_t *val)
 {
+  lsm6dsr_reg_t reg[2];
   lsm6dsr_fifo_status1_t fifo_status1;
   lsm6dsr_fifo_status2_t fifo_status2;

```

Comment:
pls remove the two lines

Reply:
They are gone, and rename the "reg" for clarity :+1: